Dockerfile

```diff
@@ -1,8 +1,13 @@
 ARG ALPINE_VERSION=latest
 FROM alpine:${ALPINE_VERSION}
 RUN apk add alpine-sdk build-base apk-tools alpine-conf busybox \
-  fakeroot syslinux xorriso squashfs-tools sudo \
+  fakeroot xorriso squashfs-tools sudo \
   mtools dosfstools grub-efi
+
+# syslinux is missing for aarch64
+ARG ARCH=amd64
+RUN if [ "${ARCH}" = "amd64" ]; then apk add syslinux; fi
```

Comment:
Can we use predefined `$TARGETARCH`?
https://docs.docker.com/engine/reference/builder/#automatic-platform-args-in-the-global-scope